test/verifiedTests/analysis/testExploration/testFindRxnsFromGenes.m

```diff
@@ -0,0 +1,53 @@
+% The COBRAToolbox: testFindRxnsFromGenes.m
+%
+% Purpose:
+%     - tests that reactions are found when providing a list of genes.
+%
+% Authors:
+%     - Original file: Stefania Magnusdottir August 2017
+%
+
+% save the current path
+currentDir = pwd;
+
+% initialize the test
+fileDir = fileparts(which('testFindRxnsFromGenes'));
+cd(fileDir);
+
+% load model
+load('Test_EcoliCoreModel.mat')
```

Comment:
What has changed between `Test_EcoliCoreModel.mat`and `ecoli_core_model`. If we keep this model, may we name it `testData_ecoli_core_model.mat`?